src/Stripe.net/Entities/PaymentIntents/PaymentIntentAmountDetailsTip.cs

```diff
@@ -9,6 +9,6 @@ public class PaymentIntentAmountDetailsTip : StripeEntity<PaymentIntentAmountDet
         /// Portion of the amount that corresponds to a tip.
         /// </summary>
         [JsonProperty("amount")]
-        public long? Amount { get; set; }
+        public long Amount { get; set; }
```

Comment:
Have we discussed this?

Reply:
> It's possible there is more changes than API changes because of our changes to Options file generation. Let me double check.

Scratch that, only changes in src/Stripe.net/Services/Sources/SourceCardOptions.cs are related to that. It looks good outside of that Amount field.

Reply:
Good catch. Fixed!